vector/src/main/java/im/vector/app/core/epoxy/bottomsheet/BottomSheetMessagePreviewItem.kt

```diff
@@ -87,6 +97,20 @@ abstract class BottomSheetMessagePreviewItem : VectorEpoxyModel<BottomSheetMessa
         holder.bodyDetails.setTextOrHide(bodyDetails?.charSequence)
         body.charSequence.findPillsAndProcess(coroutineScope) { it.bind(holder.body) }
         holder.timestamp.setTextOrHide(time)
+
+        holder.mapView.isVisible = locationData != null
+        holder.body.isVisible = locationData == null
+        locationData?.let { location ->
+            holder.mapView.initialize(object : VectorMapListener {
+                override fun onMapReady() {
+                    holder.mapView.zoomToLocation(location.latitude, location.longitude, 15.0)
```

Comment:
You should check the Fragment lifecycle state. If the Fragment is detached, it will crash.

Reply:
Done by checking `isAttachedToWindow`, hopefully enough?